cmd/buildah/commit.go

```diff
@@ -84,8 +88,11 @@ func commitListFlagSet(cmd *cobra.Command, opts *commitInputOptions) {
 	flags.IntSliceVar(&opts.encryptLayers, "encrypt-layer", nil, "layers to encrypt, 0-indexed layer indices with support for negative indexing (e.g. 0 is the first layer, -1 is the last layer). If not defined, will encrypt all layers if encryption-key flag is specified")
 	_ = cmd.RegisterFlagCompletionFunc("encryption-key", completion.AutocompleteNone)
 
+	flags.StringArrayVarP(&opts.changes, "change", "c", nil, "apply containerfile `instruction`s to the committed image")
```

Comment:
nit of a nit, but is the backtick after "instruction" rather than "instructions" intentional?

Reply:
Yes.  The argument for the flag is a single instruction, and the backticks mark for cobra what word to use as a placeholder for the argument value in the help output.